UEFIExtract/ffsdumper.cpp

```diff
@@ -15,22 +15,52 @@ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
 
 #include <fstream>
 
+static void FixFileName(UString &name)
```

Comment:
Please make it into FfsDumper::fixFileName.

Reply:
fixed in new commit.